src/main/java/dk/statsbiblioteket/doms/licensemodule/service/LicenseModuleResource.java

```diff
@@ -104,21 +104,11 @@ public ValidateAccessOutputDTO validateAccess(ValidateAccessInputDTO input)
 	}		
 					
 	
+
 	@POST	
 	@Path("getUserLicenses")	
-	@Consumes(MediaType.APPLICATION_JSON)	
-	@Produces(MediaType.APPLICATION_JSON)
-	public GetUsersLicensesOutputDTO getUserLicensesJSON(GetUsersLicensesInputDTO input)
-			                        throws LicenseModuleServiceException  {        			
-	    MonitorCache.registerNewRestMethodCall("getUserLicensesJSON");
-	    log.info("getUserLicensesJSON called");
-		return getUserLicenses(input);
-	}		
-	
-	@POST	
-	@Path("getUserLicenses")	
-	@Consumes(MediaType.TEXT_XML)	
-	@Produces(MediaType.TEXT_XML)
+	@Consumes({MediaType.TEXT_XML,MediaType.APPLICATION_JSON})	
+	@Produces({MediaType.TEXT_XML,MediaType.APPLICATION_JSON})
```

Comment:
Same as the comment pertaining to line 59 and 60

Reply:
Done